apps/server/src/clients/postgres.ts

```diff
@@ -0,0 +1,23 @@
+import { drizzle } from 'drizzle-orm/node-postgres';
+import { Pool } from 'pg';
+
+const { DB_HOST, DB_PORT, DB_USER, DB_PASSWORD, DB_NAME } = process.env;
+
+export const client = new Pool({
+    host: DB_HOST,
+    port: +DB_PORT!,
```

Comment:
Why do you need casting and null coercion here? Your env.d.ts file already handles its job.